notification/src/utils/commons.js

```diff
@@ -0,0 +1,23 @@
+function collectRequestData (request, response) {
+  return new Promise((resolve) => {
+    const data = []
+
+    request.on('data', (chunk) => {
+      data.push(chunk)
+    })
+
+    request.on('end', () => {
+      resolve(data)
+    })
+  })
+}
+
+function sendResponse (response, statusCode = 200, headers, data) {
```

Comment:
Nitpick: in such cases (with potentially many arguments), it's better to use named arguments, so you pass an object instead of positional arguments.

In this case I would have `sendResponse(response, { statusCode = 200, headers, data })`

Reply:
I'm usually doing it when it's over 6 arguments. Any particular reason for the number 4? slightly_smiling_face

Reply:
created an issue https://github.com/commercetools/commercetools-adyen-integration/issues/39